receiver/httpcheckreceiver/README.md

```diff
@@ -40,9 +40,13 @@ The following configuration settings are optional:
 ```yaml
 receivers:
   httpcheck:
-    endpoint: http://endpoint:80
-    method: GET
-    collection_interval: 10s
+    targets:
+      - endpoint: http://endpoint:80
+        method: GET
+        collection_interval: 10s
```

Comment:
Doesn't the `collection_interval` still belong to the top-level configuration? Given that the `scraperhelper.ScraperControllerSettings` is part of top-level `Config` struct, not the `targetConfig` struct.

Reply:
You are correct. Thanks for catching it.